### PR TITLE
Submarine paint setup

### PR DESCRIPTION
**Pull request by zsilencer**
submarine

### CODE REVIEW

File: src/interface/viewport.c [hunk 1]
```diff
@@ -1101,6 +1101,80 @@ bool sub_98197C(
...
+	sint8 offset_x;
+	sint8 offset_y;
+	sint8 offset_z;
+	uint8 length_x;
+	uint8 length_y;
+	uint8 length_z;
+} vehicle_boundbox;
+
+vehicle_boundbox* _submarine_vehicle_boundboxes[] = {
+	RCT2_ADDRESS(0x99123C, vehicle_boundbox),
```
Comment (duncanspumpkin) [inline src/interface/viewport.c]: Can you change this into the actual values

**Comment by duncanspumpkin** [comment]
Looks good. May as well dump the boundbox constants then there is slightly less dependencies.

**Comment by janisozaur** [comment]
This has conflicts now and needs rebasing.

**Comment by zsilencer** [comment]
Should be good now

**Pull request merged**